src/shared/ui/link-custom/link-custom.module.scss

```diff
@@ -14,110 +14,73 @@
   white-space: pre;
 }
 
-.outlined {
-  margin: 24px 0;
+.secondary-square {
+  width: max-content;
+  padding: 12px 24px;
+
+  font-size: 18px;
   color: $color-black;
+
   border: 2px solid $color-black;
 
   img {
     filter: invert(100%);
   }
 
-  &.large {
-    width: max-content;
-    padding: 12px 24px;
-    font-size: 18px;
+  &:hover {
+    background-color: $color-gray-100;
+  }
+}
 
-    &:hover {
-      background-color: $color-gray-100;
-    }
+.primary-square {
+  overflow: visible;
 
-    &.rounded {
-      width: fit-content;
-      margin: 0;
-      padding: 8px 16px;
+  width: min-content;
+  padding: 16px 24px;
 
-      color: $color-white;
+  font-size: 18px;
+  line-height: 24px;
+  color: $color-white;
 
-      border: 2px solid $color-yellow-50;
-      border-radius: 24px;
+  background-color: $color-black;
 
-      &:hover {
-        color: $color-gray-600;
-        background-color: $color-white;
-        box-shadow: 0 0 16px 0 rgba(from rgb(114 92 19) r g b / $opacity-8);
-      }
-    }
+  @include media-tablet {
+    width: 100%;
   }
 
-  &.medium {
-    @include media-mobile-landscape {
-      width: 100%;
-    }
-
-    width: 240px;
-    padding: 8px 24px;
-    font-size: 16px;
-    line-height: 24px;
-
-    &:hover {
-      background-color: $color-gray-200;
-    }
+  &:hover {
+    background-color: $color-gray-600;
   }
 }
 
-.colored {
-  width: min-content;
-
-  &.large {
-    @include media-tablet {
-      width: 100%;
-    }
+.rounded {
+  overflow: hidden;
 
-    overflow: visible;
+  width: min-content;
+  height: 36px;
+  padding: 10px 12px;
 
-    margin-top: 24px;
-    padding: 16px 24px;
+  font-size: 14px;
+  line-height: 16px;
+  color: $color-black;
 
-    font-size: 18px;
-    line-height: 24px;
-    color: $color-white;
+  background-color: #eee;
+  border-radius: 9999px;
 
-    background-color: $color-black;
+  @include media-tablet {
+    gap: 0;
```

Comment:
Change to 4px

Reply:
I use 4px here as this gap for smaller screens. Previously it was 0px, now 4px. 8px too much for this button on small screen as for me.

Reply:
@alextes90 @SpaNb4 
In my opinion, since the padding in the rounded version is smaller, then the gap needs to be smaller.
4px is a good choice in this case.